qiskit/providers/ibmq/ibmqsingleprovider.py

```diff
@@ -89,16 +89,11 @@ def _authenticate(self, credentials):
         if credentials.websocket_url:
             config_dict['websocket_url'] = credentials.websocket_url
 
-        # Prepare the version_config for the version detector.
-        version_config = config_dict.copy()
-        # By passing "access_token" in the dict, we bypass the login.
-        version_config['access_token'] = 'version_check'
-        version_connector = IBMQConnector(None, version_config,
-                                          credentials.verify)
+        version_finder = IBMQVersionFinder(auth_url=credentials.url)
```

Comment:
Just a note to remember to tweak the comments at the beginning of the method (old lines 80-83) accordingly.